util/util.go

```diff
@@ -18,3 +22,16 @@ func IsNumber(s string) bool {
 
 	return err == nil
 }
+
+// ExpandPath (path) resolves leading "~/" to user's HomeDir
+// returns expanded path, error
+func ExpandPath(path string) (string, error) {
```

Comment:
this function should be tested at some point -- I'll merge for now but this is something to keep in mind :)